anabot/runtime/installation/hub/software_selection.py

```diff
@@ -216,3 +223,49 @@ def addon_check(element, app_node, local_node):
     if action_result(element)[0] == False:
         return action_result(element)
     return addon_handler_manipulate(element, app_node, local_node, True)
+
+def get_full_kernel_option_name(kernel_option):
+    if kernel_option == '4k':
+        return FULL_4k_KERNEL_OPTION
+    elif kernel_option == '64k':
+        return FULL_64k_KERNEL_OPTION
+    else:
+        return None
+
+@handle_act('/kernel_options')
+def kernel_options_handler(element, app_node, local_node):
+    chosen_kernel_option = get_attr(element, "value")
+    if chosen_kernel_option is None:
+        return (False, "No kernel options specified")
```

Comment:
Please use `NotFoundResult` rather than a tuple (technically it is possible to use a tuple, but it's advisable to use result classes based on `ActionResult` in new code).

Reply:
This is also valid for other occurences below.

Reply:
Thanks for the suggestion! I have only seen the tuple used, so I was unaware that there was any other option :)